src/testdir/Make_all.mak

```diff
@@ -283,7 +283,8 @@ NEW_TESTS = \
 	test_xxd \
 	test_alot_latin \
 	test_alot_utf8 \
-	test_alot
+	test_alot \
+	test_string_interpolation
```

Comment:
This list should be in the alphabetical order (except the last `test_alot` tests).

Reply:
I'll fix this :+1: